data_structures/lowest_common_ancestor.py

```diff
@@ -0,0 +1,118 @@
+# Find the lowest common ancestor. Given a Binary tree and two nodes
+
+# Node for a binary tree
+class Node:
+    """Node of the binary tree."""
+
+    def __init__(self, data, left=None, right=None):
+        self.data = data
+        self.left = left
+        self.right = right
+
+    def __str__(self) -> str:
+        return "({})".format(self.data)
+
+
+class BinaryTree:
+    """Binary tress."""
```

Comment:
There is no point in this comment. It doesn't tell the user anything useful.

Reply:
Added class at the end, indicating it a binary tree class